src/main/java/taxi/controller/AddCarController.java

```diff
@@ -0,0 +1,37 @@
+package taxi.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.model.Manufacturer;
+import taxi.service.CarService;
+import taxi.service.ManufacturerService;
+
+public class AddCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final CarService carService = (CarService) injector
+            .getInstance(CarService.class);
+    private final ManufacturerService manufacturerService = (ManufacturerService) injector
+            .getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/car/add.jsp").forward(req,resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/car/add.jsp").forward(req,resp);
```
let's put all the JSPs into `views` folder

Reply:
Done, after all I was a bit confused where they all should be. Thank you for clarification.